edsl.md

```diff
@@ -340,38 +346,25 @@ Specifically, `#hashedLocation` is defined as follows, capturing the storage lay
 
     syntax Int ::= keccakIntList( IntList ) [function]
  // --------------------------------------------------
-    rule [keccakIntList]: keccakIntList(VS) => keccak(intList2ByteStack(VS))
+    rule [keccakIntList]: keccakIntList(VS) => keccak(intList2ByteArray(VS)) [concrete]
 
-    syntax ByteArray ::= intList2ByteStack( IntList ) [function]
+    syntax ByteArray ::= intList2ByteArray( IntList ) [function]
  // ------------------------------------------------------------
-    rule intList2ByteStack(.IntList) => .WordStack
-    rule intList2ByteStack(V VS)     => #padToWidth(32, #asByteStack(V)) ++ intList2ByteStack(VS)
+    rule intList2ByteArray(.IntList) => .ByteArray
+    rule intList2ByteArray(V VS)     => #buf(32, V) ++ intList2ByteArray(VS)
       requires 0 <=Int V andBool V <Int pow256
-
-    syntax IntList ::= byteStack2IntList ( WordStack )       [function]
```

Comment:
why this part is removed in edsl.md? I don't have this part in edsl2.md because I haven't used it in the specs I have proved so far. I'm not sure if they will be used in other specs.

Reply:
Please put these back if possible.

Reply:
I've put it back.